src/GeneralStateTestsFiller/stCodeSizeLimit/codesizeOOGInvalidSizeFiller.json

```diff
@@ -33,8 +33,11 @@
             }
         },
         "transaction" : {
+            "// code size is 0x600d" :" mstore(0x40, 0x60) codecopy(0x00, 0x0012, 0x600d) return(0x00, 0x600d)",
```

Comment:
Note: it's rather pseudocode than exact disassembly, it actually uses DUP1 for code size constant.
(maybe would make sense to just replace with yul?)

Reply:
Also I didn't get why MSTORE is needed, probably it's not.

Reply:
Just checked and indeed the MSTORE is not required.